### PR TITLE
feat: diagram and elf path inputs

### PR DESCRIPTION
**Pull request by leon0399**

### CODE REVIEW

**Comment by leon0399** [comment]
Example of working without adding new params: https://github.com/senseshift/senseshift-firmware/pull/127

File: action.yml [hunk 1]
```diff
@@ -42,6 +50,8 @@ runs:
       shell: bash
     - run: |
         wokwi-cli --timeout "${{ inputs.timeout }}" --expect-text "$CI_EXPECT_TEXT" \
+          ${{ inputs.scenario != '' && format('--scenario "{0}"', inputs.scenario) }} \
```
Comment (urish) [inline action.yml:53]: I guess this should be elf instead of scenario?

Comment (leon0399) [reply action.yml:53]: Yeah, I don't know what was going on inside my head 😅

**Comment by urish** [comment]
Thanks for the PR!

Unfortunately, this seems to break, as the conditionals evaluate to the string "false" when the value is not defined, e.g.:

https://github.com/wokwi/esp32-idf-hello-wifi/actions/runs/10142284968/job/28041271089

**Pull request merged**